test/integration/patients/worklist/schedule.js

```diff
@@ -1705,10 +1705,27 @@ context('schedule page', function() {
   });
 
   specify('bulk editing with work:team:manage permission', function() {
+    const testCurrentClinician = getCurrentClinician({
+      relationships: {
+        role: getRelationship(roleTeamEmployee),
+        team: getRelationship(teamNurse),
+      },
+    });
+
+    const testNonTeamMemberClincian = getClinician({
+      id: '3',
```

Comment:
Leaving these and others in this test. I'm hoping we can just remove those as part of this work: https://app.shortcut.com/roundingwell/story/54617

Reply:
agreed though this could be `uuid()` or no?

Reply:
Yes, I switched it to a `uuid()`.